### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by stepband**

### CODE REVIEW

File: src/herbivoresAndCarnivores.js [hunk 1]
```diff
@@ -1,15 +1,39 @@
...
+      Animal.alive = [];
+    }
+  }
+}
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+    Animal.alive.push(this);
```
Comment (vkryvytskyy) [inline src/herbivoresAndCarnivores.js:17]: move this to the Animal constructor

File: src/herbivoresAndCarnivores.js [hunk 2]
```diff
@@ -1,15 +1,39 @@
...
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    Animal.alive.push(this);
+  }
```
Comment (vkryvytskyy) [inline src/herbivoresAndCarnivores.js:29]: remove this

File: src/herbivoresAndCarnivores.js [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
+
+  bite(target) {
+    if (target.hidden === false) {
```
Comment (vkryvytskyy) [inline src/herbivoresAndCarnivores.js:32]: there should also be `instanceof` check

**Review by vkryvytskyy — CHANGES_REQUESTED** [review]

File: src/herbivoresAndCarnivores.js (hunk 3 above)
Comment (stepband) [reply src/herbivoresAndCarnivores.js:32]: Why I need the check here?

Comment (stepband) [reply src/herbivoresAndCarnivores.js:32]: I understand now. never mind)

**Comment by stepband** [comment]
I have made changes

**Pull request closed without merging**